README.md

```diff
@@ -1,3 +1,18 @@
+# Actions - GitHub Actions for Azure
+
+This repository provides a framework, guidleines and processes to author new and contribute to existing GitHub Actions deploying to Azure.
+
+## Table of contents
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- [Existing GitHub Actions for Azure](#existing-github-actions-for-azure)

```

Comment:
Can U just remove this line item? IMO the previous one is just enough

Reply:
Makes sense